src/simularium/SelectionInterface.ts

```diff
@@ -1,5 +1,17 @@
+import { filter } from "lodash";
+import { Color } from "three";
+import { VisGeometry } from ".";
 import { EncodedTypeMapping } from "./types";
 
+// interface for testing function without
+// having to mock the whole VisGeometry class
+export interface VisGeometryMock {
```

Comment:
this really should live closer to the tests, or it should be renamed to something like ColorIndexMap